eng/ProjectReferences.props

```diff
@@ -119,6 +119,7 @@
     <ProjectReferenceProvider Include="Microsoft.AspNetCore.Mvc" ProjectPath="$(RepositoryRoot)src\Mvc\src\Microsoft.AspNetCore.Mvc\Microsoft.AspNetCore.Mvc.csproj" />
     <ProjectReferenceProvider Include="Microsoft.AspNetCore.Mvc.TestCommon" ProjectPath="$(RepositoryRoot)src\Mvc\test\Microsoft.AspNetCore.Mvc.TestCommon\Microsoft.AspNetCore.Mvc.TestCommon.csproj" />
     <ProjectReferenceProvider Include="Microsoft.AspNetCore.Mvc.TestDiagnosticListener" ProjectPath="$(RepositoryRoot)src\Mvc\test\Microsoft.AspNetCore.Mvc.TestDiagnosticListener\Microsoft.AspNetCore.Mvc.TestDiagnosticListener.csproj" />
+    <ProjectReferenceProvider Include="Microsoft.AspNetCore.Mvc.Razor.ViewCompilation" ProjectPath="$(RepositoryRoot)src\Mvc\ViewCompilation\ViewCompilation\src\Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.csproj" />
```

Comment:
Just looked at https://github.com/aspnet/specs/blob/master/runtime/design-notes/2018-10-03-mondo-source-org.md.
Is there a reason why ViewCompilation has the extra parent `ViewCompilation` folder?
```
src/Mvc/ViewCompilation/ViewCompilation/src/ViewCompilation.csproj
src/Mvc/ViewCompilation/ViewCompilation.Tasks/Microsoft.AspNetCore.Razor.Mvc.ViewCompilation.Tasks.csproj
```
vs
```
src/Mvc/ViewCompilation/src/ViewCompilation.csproj
src/Mvc/ViewCompilation.Tasks/Microsoft.AspNetCore.Razor.Mvc.ViewCompilation.Tasks.csproj
```

Reply:
That's a good point. The spec doesn't completely define the exact folder layout, just the principles. I think it makes sense to just have `src/Mvc/ViewCompilation/` and `src/Mvc/ViewCompilation.Tasks/` instead of adding another level of nesting. It's only 2 projects.

Reply:
Decided that his doesn't matter

Reply:
I made the recommended changes anyway.